pkg/adapterManager/parallelManager_test.go

```diff
@@ -55,12 +56,12 @@ func TestExecute(t *testing.T) {
 			{&configpb.Adapter{Name: c.name}, &configpb.Aspect{Kind: c.name}},
 		}
 
-		o, err := m.Execute(context.Background(), cfg, nil, nil)
-		if c.inErr != nil && err == nil {
-			t.Errorf("m.Execute(...) = %v; want err: %v", err, c.inErr)
+		o := m.Execute(context.Background(), cfg, nil, nil)
```

Comment:
Zack captures the rational. These are API errors, not go errors.

Having dual return values is confusing. I had some mismatch in the quota aspect manager for example. With this model, the aspect managers are responsible for producing API-level errors by creating an Output object. Any go-level error they encounter must be handled and/or and turned into an API error.